.github/monitor-typescript-errors/index.js

```diff
@@ -59,6 +59,7 @@ const runner = async () => {
 	}
 
 	if ( process.env[ 'CURRENT_BRANCH' ] === 'trunk' ) {
+		setOutput( 'comment', '' );
```

Comment:
Why is it necessary?

Reply:
Initially, I add that to fix https://github.com/woocommerce/woocommerce-blocks/actions/runs/3750578628/jobs/6370472465#step:10:7, but setting content to optional fixes that issue as well. Good catch! Updating!